packages/babel-plugin-proposal-do-expressions/test/fixtures/do-expressions/semicolons.js

```diff
@@ -0,0 +1,5 @@
+expect(do {
```

Comment:
This is not a proper `expect` usage because it does not have any matchers. See https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/valid-expect.md

I guess you intended to
```js
expect(do { 1;; }).toBe(1);
expect(do { ;; }).toBe(undefined);
// ...
```